packages/host/tests/helpers/mock-matrix-service.ts

```diff
@@ -97,8 +100,8 @@ function generateMockMatrixService(
 
     profile = getMatrixProfile(this, () => this.userId);
 
-    private rooms: TrackedMap<string, Promise<RoomField>> = new TrackedMap();
-    private roomResourcesCache: Map<string, RoomResource> = new Map();
+    rooms: TrackedMap<string, RoomModel> = new TrackedMap();
+    roomResourcesCache: TrackedMap<string, RoomResource> = new TrackedMap();
```

Comment:
Having this as a non tracked map was very problematic. Particularly, we were mutating a @cached roomResources getter via roomResources.delete but we should never do so with a cached property.